app/src/main/java/nerd/tuxmobil/fahrplan/congress/models/DateInfo.kt

```diff
@@ -0,0 +1,18 @@
+package nerd.tuxmobil.fahrplan.congress.models
+
+import info.metadude.android.eventfahrplan.commons.temporal.Moment
+
+/**
+ * Represents a connection between given [date] and [dayIdx].
+ * Useful to later ask for a [dayIdx] providing a date (see [getDayIndex]).
+ * */
```

Comment:
# JavaDoc / KDoc syntax
We already taked about this - let's have it as a resolvable comment so we do not have to remember it.
The comment syntax has to be:

``` java
/**
 * Lorem ipsum
 */
```
If you search for `* */` you should find 7 matches.

Reply:
I have not worked on most comments yet ... Search an replace is a tool I know :)